package.json

```diff
@@ -10,59 +10,53 @@
     "browser-test-remote-ie": "nightwatch --config nightwatch.js --env ie",
     "browser-test-remote-safari": "nightwatch --config nightwatch.js --env safari",
     "browser-test-remote-parallel": "nightwatch --config nightwatch.js --env safari,ie,default,chrome",
-    "build": "mkdir build; browserify src/index.js -g yo-yoify -o build/app.js -t [ babelify ]; babel --plugins=transform-es2015-template-literals build/app.js --out-file build/app.js",
+    "build": "mkdirp build; browserify src/index.js -t yo-yoify -t [ babelify --presets [ es2015 ] ] | babili > build/app.js",
     "csslint": "csslint --ignore=order-alphabetical --errors='errors,duplicate-properties,empty-rules' --exclude-list='assets/css/font-awesome.min.css' assets/css/",
-    "downloadsolc": "rm soljson.js; wget https://ethereum.github.io/solc-bin/soljson.js",
-    "lint": "standard",
+    "downloadsolc": "rm -f soljson.js; wget https://ethereum.github.io/solc-bin/soljson.js",
```

Comment:
i had some problem related to deleting `soljson.js` and the `-f` flag solved it.